src/Felix/Construct/Product.agda

```diff
@@ -140,11 +141,11 @@ module product-homomorphisms where instance
   H₂ : Homomorphism _⇨_ _⇨₂_
   H₂ = record { Fₘ = proj₂ }
 
-  catH₁ : ∀ {q₁ q₂} ⦃ _ : Equivalent q₁ _⇨₁_ ⦄ ⦃ _ : Equivalent q₂ _⇨₂_ ⦄ → CategoryH _⇨_ _⇨₁_
-  catH₁ = record { F-cong = proj₁ ; F-id = refl ; F-∘ = refl }
+  catH₁ : ∀ {q₁ q₂} ⦃ eq₁ : Equivalent q₁ _⇨₁_ ⦄ ⦃ eq₂ : Equivalent q₂ _⇨₂_ ⦄ → CategoryH _⇨_ _⇨₁_
+  catH₁ ⦃ eq₁ ⦄ = record { F-cong = proj₁ ; F-id = eq₁ .refl ; F-∘ = eq₁ .refl }
```

Comment:
I don't know this postfix dot notation (`eq₁ .refl`). Where did you find it?

Reply:
Some random agda code I've seen on gh. Although I think it was behind [feature flag](https://agda.readthedocs.io/en/v2.6.4/tools/command-line-options.html#cmdoption-postfix-projections) once. I think I struggle a bit to remember the proper record access syntax now and tend to type stuff until it will work.

Reply:
Neat. Thanks!